packages/client/src/components/Players/Player.tsx

```diff
@@ -10,33 +10,25 @@ import { Box, BoxProps, Grid } from 'theme-ui';
 type Props = {
   player: PlayerState;
   isActive: boolean;
-  movingCards: string[];
   togglePlayerActiveCard: React.Dispatch<React.SetStateAction<string | null>>;
   activePlayerCard: string | null;
 };
 
-export const Player = ({
-  player,
-  isActive,
-  movingCards,
-  togglePlayerActiveCard,
-  activePlayerCard,
-  ...rest
-}: Props & BoxProps) => {
+export const Player = ({ player, isActive, togglePlayerActiveCard, activePlayerCard, ...rest }: Props & BoxProps) => {
   if (!player) {
     return <InvitePlayer />;
   }
   const { captured, hand } = player;
   return (
     <Box {...rest}>
       <Grid sx={{ m: 3 }} columns="1.5fr 1fr 1fr 1fr">
-        <Deck cardNumber={captured.length} scopa={player.scopa} />
+        <Deck cardNumber={captured.length} scopa={player.scopa} id="player-deck" />
```

Comment:
These should probably live in a constants file, right?

Reply:
yes I was thinking to do that too, but for now it's just to show off